src/Simulator/IncrementalRedstoneSimulator/PressurePlateHandler.h

```diff
@@ -92,4 +185,53 @@ class cPressurePlateHandler : public cRedstoneHandler
 		UNUSED(a_Meta);
 		return {};
 	}
+
+private:
+	static AString GetClickOnSound(BLOCKTYPE a_BlockType)
+	{
+		// manage on-sound
+		AString soundToPlay = "";
+		switch (a_BlockType)
+		{
+			case E_BLOCK_STONE_PRESSURE_PLATE:
+				soundToPlay = "block.wood_pressureplate.click_on";
```

Comment:
You can drop the `soundToPlay` variable:
```cpp
case E_BLOCK_STONE_PRESSURE_PLATE:
	return "block.wood_pressureplate.click_on";
// etc...
```

Reply:
True :D